### PR TITLE
Add a conda environment file.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

**Comment by Mr0grog** [comment]
Should this also modify the [deployment.md](https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/master/deployment.md)?

File: deployment.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ $ curl <conda_url> > conda_installer.sh
 # e.g: curl https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh > conda_installer.sh
 ```
 
-Then run the installer. Follow the prompts, but instead of the default install location, choose `/opt/conda`. (This gets us a system-wide install so the web server can use it.)
+Then run the installer. (These parameters get us a system-wide install so the web server can use it.)
 
 ```sh
-$ sudo bash conda_installer.sh
+$ sudo bash conda_installer.sh -b -p /opt/conda
```
Comment (Mr0grog) [inline deployment.md]: Ohhhh, this is much nicer!

**Comment by Mr0grog** [comment]
Catching back up on this; is it good to go? (Can you add credentials to your Circle account so these tests can pass in the future, @danielballan?)

**Comment by danielballan** [comment]
> Can you add credentials to your Circle account

Just did:

screen shot 2018-05-04 at 2 35 18 pm

And re-ran the build a couple times, but they didn't take. Maybe they only affect new PRs? But if that were true, it should would be difficult to debug issues with env vars....

My user experience with the Circle UI has been uniformly awful.

**Comment by danielballan** [comment]
Oh -- saw your comment in the review too late. See #184. Thanks.

**Pull request merged**